Solutions/SemperisDirectoryServicesProtector/Data Connectors/SemperisDSP-connector.json

```diff
@@ -0,0 +1,152 @@
+            "instructions": [  
+            ]    
+        }, 
+        {
+            "title": "1. Configure Semperis DSP Management Server to send Windows event logs to your Azure Sentinel Workspace",
+            "description": "On your **Semperis DSP Management Server** install the Microsoft agent for Windows."
+        },
+        {
+            "title": "2. Install and onboard the Microsoft agent for Windows",
+            "description": "You can skip this step if you have already installed the Microsoft agent for Windows",
+            "instructions": [
+                {
+                    "parameters": {
+                        "title": "Choose where to install the agent:",
+                        "instructionSteps": [
+                            {
+                                "title": "Install agent on Semperis DSP Management Server",
+                                "description": "Download the agent on the relevant machine and follow the instructions.",
+                                "instructions": [
+                                    {
+                                        "parameters": {
+                                            "linkType": "InstallAgentOnNonAzure"
+                                        },
+                                        "type": "InstallAgent"
+                                    }
+                                ]
+                            }
+                        ]
+                    },
+                    "type": "InstructionStepsGroup"
+                }
+            ]
+        },
+        {
+            "title": "3. Configure the Semperis DSP Windows event logs to be collected by the agent",
+            "description": "Configure the agent to collect the logs.\n\n1. Under workspace advanced settings **Configuration**, select **Data** and then **Windows Event Logs**.\n2. Select **Go to Agents configuration** and click **Add Windows event log**.\n3. Enter **Semperis-DSP-Security/Operational** as the log name to be collected and click **Apply**",
+            "instructions": [
+                {
+                    "parameters": {
+                        "linkType": "OpenAdvancedWorkspaceSettings"
+                    },
+                    "type": "InstallAgent"
+                }
+            ]
+        },
+        {
+            "title": "",
+            "description": "> You should now be able to receive logs in the *Windows event log* table, log data can be parsed using the **dsp_parser()** function, used by all query samples, workbooks and analytic templates."
+        }
+    ],
+    "metadata": {
+        "id": "TBD",
+        "version": "1.0.0",
+        "kind": "dataConnector",
+        "source": {
+            "kind": "community"
+        },
+        "author": {
+            "name": "Semperis"
+        },
+        "support": {
+            "name": "Semperis",
+            "link": "https:/https://www.semperis.com/contact-us/",
```

Comment:
Please fix the typo in the url, **https:/** is repeated in the url.

Thanks!!!

Reply:
Great catch, thank you